packages/Organisms/SiteHeader/index.vue

```diff
@@ -335,6 +349,29 @@ export default {
         width: $rpl-site-header-logo-width;
         margin-left: $rpl-site-header-menu-toggle-border-spacing;
       }
+
+      &--vic-logo-primary {
+        display: block; //always show vic.gov logo if no cobrand logo
+
+        img {
+          width: $rpl-site-header-logo-primary-width;
+        }
+
+        &--cobrand {
+
+          padding-right: 0.5rem;
+          border-right: 1px solid #fff;
+          display: none;
+
+          @include rpl_breakpoint('m'){
```

Comment:
spacing between is missing. Sorry we have a missing sass lint on the vue template. So it's not easy for you to pick those sass code standard.